src/pages/Forgotpassword/Forgotpassword.scss

```diff
@@ -1 +1,26 @@
-.Forgotpassword {}
```

Comment:
Hi Pema,
Could u please keep this global class per component as it is.. as there is no built in vie3 encapsulation vailable in react.. we have to take care of css